android/app/src/androidTest/java/com/trustwallet/core/app/utils/TestLiquidStaking.kt

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2017-2023 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+package com.trustwallet.core.app.utils
+
+import com.google.protobuf.ByteString
+import org.junit.Assert
+import wallet.core.jni.*
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.java.AnySigner
+import wallet.core.jni.proto.Ethereum
+import wallet.core.jni.proto.LiquidStaking
+import wallet.core.jni.LiquidStaking as WCLiquidStaking
+
+class TestLiquidStaking {
+    init {
+        System.loadLibrary("TrustWalletCore");
+    }
+
+    @Test
+    fun testStraderStakeMatic() {
+        val input = LiquidStaking.Input.newBuilder()
+        input.apply {
+            blockchain = LiquidStaking.Blockchain.POLYGON
+            protocol = LiquidStaking.Protocol.Strader
+            smartContractAddress = "0xfd225c9e6601c9d38d8f98d8731bf59efcf8c0e3"
+            stake = LiquidStaking.Stake.newBuilder().apply {
+                amount = "1000000000000000000"
+                asset = LiquidStaking.Asset.newBuilder().apply {
+                    stakingToken = LiquidStaking.Coin.MATIC
+                }.build()
+            }.build()
+        }
+
+
+        val inputSerialized = input.build().toByteArray()
+        assertEquals(Numeric.toHexString(inputSerialized), "0x0a170a00121331303030303030303030303030303030303030222a3078666432323563396536363031633964333864386639386438373331626635396566636638633065333001")
+        val outputData = WCLiquidStaking.buildRequest(inputSerialized)
```

Comment:
Interesting, swift code doesn't have `WC` prefix, just `LiquidStaking` not `WCLiquidStaking`